builtins/amp-img.js

```diff
@@ -198,6 +198,11 @@ export class AmpImg extends BaseElement {
     if (sizes) {
       return;
     }
+    // Auto-sizes are not compatible with intrinsic layout.
```

Comment:
Nit: refer to the original issue for context in this comment, since it isn't immediately obvious why these two are incompatible. 

The reason why intrinsic layout is not compatible with auto-generating sizes is that by definition, the image is intended to grow to its natural size. However, due to the cache autogenerating `srcset` and the runtime (the code that we are modifying here) auto-generating sizes, it may change the natural size of the image.